src/picosvg/svg.py

```diff
@@ -371,6 +371,7 @@ def _inherit_nondefault_overflow(attrib, child, attr_name):
             "fill": _inherit_copy,
             "fill-rule": _inherit_copy,
             "style": _inherit_copy,
+            "transform": _inherit_copy,
```

Comment:
it's not sufficient to inherit and copy the transform attribute from parent group element to child element like you do here; we need an `_inherit_matrix_multiply` that calls `Affine2D.product(child_transform, group_transform)`, similar to how we do inside `apply_transforms`.

Reply:
I pushed bc4fbf8 to fix this, PTAL

Reply:
Oops, that was ... let's politely say silly. Ty for the fix :)